src/bench/bench_java/n5_benchmark.py

```diff
@@ -22,15 +17,8 @@ def set_up():
         rmtree(BENCH_DIR)
     os.mkdir(BENCH_DIR)
 
-    im_file = os.path.join(BENCH_DIR, 'im.zip')
-    call(['wget', '-O', im_file, IM_URL])
-
-    with zipfile.ZipFile(im_file) as f:
-        f.extract('JeffT1_le.tif', BENCH_DIR)
-
-    im_file = os.path.join(BENCH_DIR, 'JeffT1_le.tif')
-    im = np.array(io.imread(im_file))
-    return im[30:94, 100:164, 100:164].astype('uint8')
+    im = z5py.util.fetch_test_data()
```

Comment:
Util is not imported in the z5py `__init__.py`, so this will fail.
I would vote for keeping it this way and `import z5py.util` or `from z5py.util import fetch_test_data` at the top of the file.
Same for other instances below.